integrations/kube-agent-updater/Dockerfile

```diff
@@ -25,7 +25,7 @@ ARG TARGETARCH
 # Build the program. We rely on golang's cross-compilation capabilities for multiarch building.
 RUN echo "Targeting $TARGETOS/$TARGETARCH" && \
     GOOS=$TARGETOS GOARCH=$TARGETARCH CGO_ENABLED=0 \
-    go build -a -o /go/bin/teleport-kube-agent-updater github.com/gravitational/teleport/integrations/kube-agent-updater/cmd/teleport-kube-agent-updater
+    go build -a -ldflags "-w -s" -o /go/bin/teleport-kube-agent-updater github.com/gravitational/teleport/integrations/kube-agent-updater/cmd/teleport-kube-agent-updater
```

Comment:
Is this change intended to be in this PR ?

Reply:
yes but I reverted it, I will consolidate this later